### PR TITLE
return $this from setters to allow for method chaining.

### PR DESCRIPTION
**Pull request by simplechris**
Allow for method chaining, and more of a fluent interface when writing `Query` classes etc.

for example:

``` php
$query = $this->get('widop_google_analytics.query')
    ->setIds('ga:XXXXXXXX')
    ->setStartDate(new \DateTime('2012-01-01'))
    ->setEndDate(new \DateTime());
```

Perhaps this is just my personal preference, but feel free to merge :thumbsup:

### CODE REVIEW

**Comment by clemherreman** [comment]
I personally like it this way, but if I recall, @egeloen specifically removed the fluent interface a while ago.

File: Model/Client.php [hunk 1]
```diff
@@ -66,10 +66,12 @@ public function getClientId()
      * Sets the client ID.
      *
      * @param string $clientId The client ID.
+     * @return $this
      */
     public function setClientId($clientId)
     {
         $this->clientId = $clientId;
+        return $this;
```
Comment (blazarecki) [inline Model/Client.php:76]: In this bundle we use the [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md).

Here there is a missing blank line before the return statement. (Same for the others)

File: Model/Client.php [hunk 2]
```diff
@@ -66,10 +66,12 @@ public function getClientId()
      * Sets the client ID.
      *
      * @param string $clientId The client ID.
+     * @return $this
```
Comment (blazarecki) [inline Model/Client.php]: Here there is a missing blank line before the return tag and you should use the FQCN

``` php
@return \Widop\GoogleAnalyticsBundle\Model\Client The client.
```

Same for the others (Don't forget to change the FQCN :))

**Comment by blazarecki** [comment]
@egeloen has agreed about this change.

:+1:

File: Model/Client.php [hunk 3]
```diff
@@ -66,10 +66,13 @@ public function getClientId()
      * Sets the client ID.
      *
      * @param string $clientId The client ID.
+     * @return \Widop\GoogleAnalyticsBundle\Model\Client The client.
```
Comment (egeloen) [inline Model/Client.php:70]: Sorry guy but you have forgotten to add a blank line before each return statement in the PHPDoc. Can you fix it plz? Then I will merge your PR. Thx for your contribution.

Comment (simplechris) [inline Model/Client.php:70]: no problem, I've rebased and squashed my changes into a single commit here: a29deb9 
:thumbsup:

**Comment by simplechris** [comment]
Cool.
I've rebased and squashed my changes into a single commit: a29deb9
:thumbsup:

**Comment by egeloen** [comment]
Good to merge :+1:

**Pull request merged**